packages/flutter/lib/src/services/platform_views.dart

```diff
@@ -484,13 +637,25 @@ class AndroidViewController {
   static const int kAndroidLayoutDirectionRtl = 1;
 
   /// The unique identifier of the Android view controlled by this controller.
-  final int id;
+  @override
+  final int viewId;
 
   final String _viewType;
 
   /// The texture entry id into which the Android view is rendered.
   int _textureId;
 
+  // Helps convert PointerEvents to AndroidMotionEvents.
+  final _AndroidMotionEventConverter _motionEventConverter =
+    _AndroidMotionEventConverter();
```

Comment:
What do you think about moving `_AndroidMotionEventConverter` methods to this controller?

Reply:
I think it makes more sense to separate them. This class has to maintain a state and record `PointerEvent`s. I felt the logic was easier to follow if all of this was in it's own class.